Doc/library/test.rst

```diff
@@ -1634,3 +1452,197 @@ The :mod:`test.support.threading_helper` module provides support for threading t
        # (to avoid reference cycles)
 
    .. versionadded:: 3.8
+
+
+:mod:`test.support.os_helper` --- Utilities for os tests
+========================================================================
+
+.. module:: test.support.os_helper
+   :synopsis: Support for os tests.
+
+The :mod:`test.support.os_helper` module provides support for os tests.
+
+.. versionadded:: 3.10
+
+
+.. data:: FS_NONASCII
+
+   A non-ASCII character encodable by :func:`os.fsencode`.
+
+
+.. data:: SAVEDCWD
+
+   Set to :func:`os.getcwd`.
```

Comment:
Side note: I'm not sure why it exists in test.support. It seems like only test.libregrtest uses it. Maybe it should be moved (later) to test.libregrtest.

Reply:
Hm, I checked this detail. Looks like the describe info is right. It points that `SAVEDCWD=os.getcwd()`